cengage/react-magma
# feat(Stepper): Additions to the Stepper component for accessibility compliance. The 'aria-current' attribute now reads 'step' when active, the DOM structure has been changed to a UL LI layout, and when labels are hidden, the screenreader sees the step state for each step label.

Issue: # [1143](https://github.com/cengage/react-magma/issues/1143)

## What I did
- Changed aria-current to read "step" when active.
- Converted to UL LI structure for the component layout.
- Added the state for each step when labels are hidden.

## Checklist 
- [N/A] I have made corresponding changes to the documentation
- [N/A] I have added tests that prove my fix is effective or that my feature works

## How to test
- Make sure aria-current reads "step".
- Verify in DOM that the UL LI structure looks sufficient.
- Ensure screen reader can read the step state after the label when labels are hidden.
